### DISCUSSION TITLE
How to use scaling factor when training a model？

### DISCUSSION THREAD

**Discussion by Qianruipku** [deepmodeling/deepmd-kit#2553] [Q&A]
In ref https://pubs.aip.org/aip/pop/article/27/12/122704/108206/Warm-dense-matter-simulation-via-electron, scaling factor is introduced to treate temperatures that span several orders of magnitude. 
The loss function is weighted by this factor:
![image](https://github.com/deepmodeling/deepmd-kit/assets/76200646/3e44ecb1-83ad-46c9-8f20-d32d7fad9530)

I wonder if there is a frame parameter that exists both in the fitting net and in the defination of loss function (or any parameter only controls the weight of the loss function) and how to use it when training a DP model and running Lammps.
(Currently, to my understanding, the fparam parameter only exists in the fitting network?)
![image](https://github.com/deepmodeling/deepmd-kit/assets/76200646/f7e37071-d8f3-4308-842b-fa0888083348)

**Comment by wanghan-iapcm** [comment]
It is not supported in the deepmd-kit, you need some hack of the source code. You are welcome to contribute it in the package. @AnguseZhang FYI
